stock_picking_transfer_lot_autoassign/models/stock_picking.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.model
```

Comment:
Even if it was done like this in v9 module, we cannot do that as there is an inconsistency between self value (which is a single recordset) and the 'model' decorator.

I think we don't have to decorate (like in stock module)

Reply:
This has been perverted in the migration. In v9 the decorator has all the sense due to standard signature:

https://github.com/odoo/odoo/blob/7eb7e9fd6e72764f28df19ab8803e1cceff38b4d/addons/stock/stock.py#L1204

And the inherited method adapted to new API:

https://github.com/OCA/stock-logistics-workflow/blob/c466ffc62170b6b28f059e6ad6fba85624f8bbaf/stock_picking_transfer_lot_autoassign/models/stock_picking.py#L12

with `picking` as the argument and without using `self`

Reply:
Ok, so no decorator here.

Reply:
Yes, as `@api.multi` is now the default.

Reply:
Be careful that in the super, Odoo consider self as a single recordset